src/apps/vpn/mozillavpn.cpp

```diff
@@ -1662,6 +1665,20 @@ void MozillaVPN::scheduleRefreshDataTasks(bool refreshProducts) {
           TaskGetSubscriptionDetails::NoAuthenticationFlow,
           ErrorHandler::PropagateError)};
 
+  // The VPN needs to be off in order to determine the client's real location.
+  // And it also needs to complete before TaskServers in case this triggers an
+  // automatic server selection.
+  //
+  // TODO: This ordering requirement can be relaxed in the future once automatic
```

Comment:
do you want to file a jira issue to keep track of it?

Reply:
I should!